Module#02/service/src/main/java/com/epam/esm/service/DefaultCertificateService.java

```diff
@@ -0,0 +1,81 @@
+package com.epam.esm.service;
+
+import com.epam.esm.criteria.Criteria;
+import com.epam.esm.dao.CertificateDao;
+import com.epam.esm.dto.CertificateDto;
+import com.epam.esm.dto.CertificateWithoutTagDto;
+import com.epam.esm.exception.CertificateIsExistsException;
+import com.epam.esm.exception.CertificateNotFoundException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+import static com.epam.esm.mapper.CertificateMapper.mapper;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class DefaultCertificateService implements CertificateService {
```

Comment:
Default is not good name. CertificateServiceImp will be enough

Reply:
changed the name of Default to Impl